extensions/cms/package.nls.json

```diff
@@ -2,14 +2,13 @@
 	"cms.displayName": "CMS",
 	"cms.description": "Central Management Servers",
 	"cms.title": "Central Management Servers",
-	"cms.connectionProvider.displayName": "Microsoft SQL Server - CMS",
+	"cms.connectionProvider.displayName": "Microsoft SQL Server",
 	"cms.resource.explorer.title": "Central Management Servers",
 	"cms.resource.refreshServerGroup.title": "Refresh Server Group",
-	"cms.resource.refreshCmsServer.title": "Refresh Central Management Server",
 	"cms.resource.deleteRegisteredServer.title": "Delete Registered Server",
-	"cms.resource.addRegisteredServer.title": "Add Registered Server",
+	"cms.resource.addRegisteredServer.title": "New Server Registration...",
 	"cms.resource.deleteServerGroup.title": "Delete Server Group",
-	"cms.resource.addServerGroup.title": "Add Server Group",
+	"cms.resource.addServerGroup.title": "New Server Group...",
 	"cms.resource.registerCmsServer.title": "Add Central Management Server",
```

Comment:
>Add [](start = 42, length = 3)

New CMS? To keep it in line with the other changes?

Reply:
I used all the strings from SSMS, except `Register Central Management Server...` which was given to me during the PM meeting.